pwndbg/lib/cache.py

```diff
@@ -0,0 +1,160 @@
+"""
+Caches return values until some event in the inferior happens,
+e.g. execution stops because of a SIGINT or breakpoint, or a
+new library/objfile are loaded, etc.
+"""
+
+from collections import UserDict
+from functools import wraps
+from typing import Any
+from typing import Callable
+from typing import Dict
+from typing import Tuple
+
+# Set to enable print logging of cache hits/misses/clears
+NO_DEBUG, DEBUG_GET, DEBUG_CLEAR, DEBUG_SET = 0, 1, 2, 4
```

Comment:
A [Flag](https://docs.python.org/3/library/enum.html#enum.Flag) might make sense here instead.

Reply:
I think this is so simple it can stay as it is :P